src/core_plugins/kibana/public/dashboard/directives/dashboard_panel.js

```diff
@@ -1,15 +1,19 @@
 import _ from 'lodash';
 import 'ui/visualize';
 import 'ui/doc_table';
-import { loadPanelProvider } from 'plugins/kibana/dashboard/components/panel/lib/load_panel';
+import 'plugins/kibana/dashboard/services/get_object_loaders_for_dashboard.factory';
```

Comment:
This was motivated by some conversations with @w33ble and a new style pattern the Management team has been experimenting with.  This morning I decided I'm on the fence with whether or not I should be incorporating this new pattern, read more here: https://github.com/elastic/kibana/pull/9859#issuecomment-272880859

Once I hear back from @w33ble and @cjcenizal on that comment I'll decide whether to revert the `.factory` naming scheme.

FWIW I do like the pattern a lot but since it's not fully adopted by the team, it does seem a bit premature, especially with the last email that went around about consistency trumping new patterns until a certain # of criteria is hit.